source/basic.tex

```diff
@@ -4059,8 +4058,8 @@
 \tcode{void}~(\ref{basic.types}) has three corresponding cv-qualified
 versions of its type: a \defn{const-qualified} version, a
 \defn{volatile-qualified} version, and a
-\defn{const-volatile-qualified} version. The term
-\term{object type}~(\ref{intro.object}) includes the cv-qualifiers
+\defn{const-volatile-qualified} version. An
+object's type~(\ref{intro.object}) includes the cv-qualifiers
```

Comment:
Here, perhaps "The type of an object" instead of "An object's type"?

While we're looking at this line, is there a reason why "cv-qualifiers" here isn't `\grammarterm`'d? We definitely seem to be talking about the grammar element.